src/clj/rems/db/applications.clj

```diff
@@ -55,7 +55,7 @@
     :items [catalogue-item-id]}))
 
 (defn- valid-user? [userid]
-  (not (nil? (users/get-user-attributes userid))))
+  (some? (users/get-user userid)))
```

Comment:
Hmm, doesn't this always return true now that users/get-user augments the return value with the userid even if the user doesn't exist?

Reply:
good catch!